src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3321,10 +3321,7 @@ Give the compiler some way to differentiate the methods. For example, you can gi
     <value>'id#' syntax is no longer supported. Use '$id' instead.</value>
   </data>
   <data name="WRN_DefineIdentifierRequired" xml:space="preserve">
-    <value>Invalid value for '/define'; '{0}' is not a valid identifier</value>
```

Comment:
Actually,the message is clearer now even for command line parser.

---
In reply to: [105717049](https://github.com/dotnet/roslyn/pull/17113#discussion_r105717049) [](ancestors = 105717049)